governance/RELEASE-MANAGEMENT.md

```diff
@@ -0,0 +1,78 @@
+# Release Management
+
+This document describes ORAS project release management, which includes release versioning, supported releases, and supported upgrades. This document is applicable to all ORAS sub-projects.
+
+## Glossary of Terms
+
+- **X.Y.Z** refers to the version (based on git tag) of ORAS sub-project that is released. 
+- **Breaking changes** refer to schema changes, flag changes, and behavior changes of ORAS sub-projects that may require existing content to be upgraded and may also introduce changes that could break backward compatibility.
+- **Milestone** GitHub milestones are used by maintainers to manage each release. PRs and Issues for each release should be created as part of a corresponding milestone.
+- **Patch releases** refer to applicable fixes, including security fixes, may be backported to support releases, depending on severity and feasibility.
+
+## Release Versioning
+
+ORAS follows the [Semantic Versioning 2.0.0](https://semver.org/) to define the release versioning. All releases will be of the form _vX.Y.Z_ where X is the major version, Y is the minor version and Z is the patch version. This project strictly follows semantic versioning.
+
+The rest of the doc will cover the release process for the following kinds of releases:
+
+### Major Releases
+
+MAJOR version should be considered when there are incompatible API changes or breaking changes.
+
+The ORAS CLI and ORAS-go project reached a stable version. Other sub-projects in Major version zero  (0.y.z) is in initial development. 
+
+### Minor Releases
+
+- **ALPHA:** X.Y.0-alpha.W, W >= 0 (Branch : main)
```

Comment:
Why `W >= 0`? The version `2.0.0-alpha.0` does not make sense to me so as `2.0.0-rc.0`.
Should it be `W > 0`?